src/Application.php

```diff
@@ -56,6 +56,16 @@ class Application extends BaseApplication implements ArrayAccess
     ];
 
     /**
+     * The application's core commands.
+     *
+     * @var array
+     */
+    protected $coreCommands = [
+        NunoMaduro\ZeroFramework\Commands\Builder::class,
+        NunoMaduro\ZeroFramework\Commands\Renamer::class,
```

Comment:
Since by doing `use NunoMaduro\ZeroFramework\Commands;` we have already imported that namespace to the current namespace, can't we remove the `Commands\` & rather just do 

```php
Builder::class,
Renamer::class,
```

Reply:
Nop. To do that you have to include both commands on the top 👎

Reply:
@nunomaduro But it works, I am using the Sublimephp companion plugin which also points that it's referencing the correct classes.

Reply:
I think it works because, as we have already imported that namespace into the current namespace by using the `use` statement, so it can now find both the classes in it.

Reply:
Weird, but ok 😊